generator/src/main/java/com/scottlogic/deg/generator/GenerationEngine.java

```diff
@@ -32,7 +34,7 @@ public GenerationEngine(
         this.decisionTreeGenerator = decisionTreeGenerator;
     }
 
-    void generateDataSet(Profile profile, GenerationConfig config) throws IOException {
+    public void generateDataSet(Profile profile, GenerationConfig config) throws IOException {
```

Comment:
Why is this now public?

Reply:
it is now being called from another package, so needed to be changed from the default `package` to `public`